bindings/gumjs/gumquickcore.c

```diff
@@ -3537,6 +3537,7 @@ GUMJS_DEFINE_CONSTRUCTOR (gumjs_native_callback_construct)
   cb->wrapper = wrapper;
   cb->func = func;
   cb->core = core;
+  cb->interceptor_replacement_count = 0;
```

Comment:
Since we're using `g_slice_new0()` here, all fields will be zero-initialized and we can omit this.